### PR TITLE
github workflows: Convert top images and check bootstrap format

### PR DESCRIPTION
**Pull request by yawqi**
Add v6 images conversion to CI, and check it with the `fsck.erofs`.

Signed-off-by: Qi Wang <wangqi@linux.alibaba.com>

### CODE REVIEW

**Comment by yawqi** [comment]
This has passed CI in my own repo, https://github.com/yawqi/image-service/actions/runs/2472519990, https://github.com/yawqi/image-service/actions/runs/2472437682.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -78,3 +82,60 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: EmbarkStudios/cargo-deny-action@v1
+
+  convert-images:
```
Comment (imeoer) [inline .github/workflows/ci.yml:86]: The job will slow down the CI run of each PR, can we run it only in daily testing (convet.yaml) ?

Comment (yawqi) [reply .github/workflows/ci.yml:86]: Thanks! OK.

Comment (yawqi) [reply .github/workflows/ci.yml:86]: Thanks! OK.

**Pull request closed without merging**